app/controllers/actors_controller.rb

```diff
@@ -0,0 +1,49 @@
+class ActorsController < ApplicationController
```

Comment:
In this controller you need to have safe check so non admins can't hit this endpoint. Could be done with controller `before_action :reject_if_not_admin`

Reply:
and then you need to test that on the integration tests